### ISSUE TITLE
Brackets color in React syntax

### ISSUE THREAD

**Issue by BrenoVelasco** [Binaryify/OneDark-Pro#358]
Good morning!
The color of brackets in props pass changed from purple to red. Is there a way to revert this?

Before:
![Screenshot from 2019-06-10 12-06-07](https://user-images.githubusercontent.com/21113340/69258103-d71b9680-0b9a-11ea-9e6b-6562adc3d7d5.png)

After:
![Screenshot from 2019-11-20 13-37-15](https://user-images.githubusercontent.com/21113340/69258133-e7337600-0b9a-11ea-9fd6-f7765ae4104e.png)

Thank you!

**Comment by Binaryify** [comment]
could you provide the code on your screenshots

**Comment by BrenoVelasco** [comment]
```<MaskedInputComponent
          handleParentState={handleChange}
          id='telefone'
          name='telefone'
          label='Telefone'
          state='telefone'
          value={telespectador.telefone}
          phoneField
          disabled={user.usuario_crc !== usuarios.CRC}
      />

<Dialog
    open={!!dialog.isOpen}
    onClose={handleClose}
    fullWidth={fullWidth}
    maxWidth={maxWidth}
    aria-labelledby={`dialog-${slug}`}
>
</Dialog>
```

**Comment by Binaryify** [comment]
just release a new version, you can check if it fixed now

**Comment by zhangbenber** [comment]
The same happened in the `${}` token inside ` `` ` template strings, even after this jsx one is fixed

**Comment by Binaryify** [comment]
@zhangbenber 
just fixed that in v3.1.2, you can check it later

**Comment by BrenoVelasco** [comment]
Thank you!
